cookbook/session/locale_sticky_session.rst

```diff
@@ -53,8 +53,8 @@ how you determine the desired locale from the request::
         public static function getSubscribedEvents()
         {
             return array(
-                // must be registered before the default Locale listener
-                KernelEvents::REQUEST => array(array('onKernelRequest', 17)),
+                // must be registered after the default Locale listener
+                KernelEvents::REQUEST => array(array('onKernelRequest', 16)),
```

Comment:
let's use 15 to be sure that it's executed after (the default listener is on 16)